zerver/tests/test_decorators.py

```diff
@@ -1134,11 +1134,11 @@ def test_authenticated_json_post_view_with_get_request(self) -> None:
             )
 
         with self.assertLogs(level="WARNING") as mock_warning:
-            result = self.client_get(r"/json/remotes/server/register", {"stream": "Verona"})
+            result = self.client_get(r"/json/subscriptions/exists", {"stream": "Verona"})
             self.assertEqual(result.status_code, 405)
             self.assertEqual(
                 mock_warning.output,
-                ["WARNING:root:Method Not Allowed (GET): /json/remotes/server/register"],
+                ["WARNING:root:Method Not Allowed (GET): /json/subscriptions/exists"],
```

Comment:
This is just replacing this endpoint with a different `/json` that's decorated by `@require_post`, which is what this test wants I believe

Reply:
This is OK for this PR, but given dcbc8e66fb146f543c1ec16d46d93b64f195c2c2, I think it's possible we actually should just delete some of these tests; @andersk am I correctly reading that we probably forgot to delete tests in that commit?